strictEquals.js

```diff
@@ -0,0 +1,4 @@
+const strictEquals = (numOne, numTwo) => {
```

Comment:
`numOne` and `numTwo` are not good names since these values could very well be strings or arrays.